python/tank/bootstrap/manager.py

```diff
@@ -143,6 +146,18 @@ def _get_pipeline_configuration(self):
         """
         return self._pipeline_configuration_identifier
 
+    def _set_allow_config_overrides(self, state):
```

Comment:
I got the go ahead from Rob to use property setters since we're dropping 2.5 by the time this code gets release. Just sayin'. ;)